exchange/exchange_test.go

```diff
@@ -1772,7 +1777,27 @@ func newExchangeForTests(t *testing.T, filename string, expectations map[string]
 		categoriesFetcher:   categoriesFetcher,
 		bidderInfo:          bidderInfos,
 		externalURL:         "http://localhost",
+		bidIDGenerator:      bidIDGenerator,
+	}
+}
+
+type mockBidIDGenerator struct {
+	GenerateBidID bool
```

Comment:
Nitpick: Please add json attributes to use `generateBidID` and `returnError` in the test files for naming consistency.